File: Demohahaha/Barista.cs
class Barista {
        static void Main(string[] args) {

        EspressoBeverage latte = new VanillaSyrup(new EspressoShot(new VanillaLatte()));

        Console.WriteLine("Ingredients: " + latte.getDescription());

        Console.WriteLine("Total Cost: " + latte.getCost());

    }
}

File: src/main/java/Barista.java
public class Barista {

    public static void main(String[] args) {

        EspressoBeverage latte = new VanillaSyrup(new EspressoShot(new VanillaLatte()));

        System.out.println("Ingredients: " + latte.getDescription());

        System.out.println("Total Cost: " + latte.getCost());

    }
}
